docs/en/Logging-Conventions.md

```diff
@@ -140,3 +140,12 @@ And the following command gets all workers' log level on class `alluxio.heartbea
 ```bash
 alluxio logLevel --logName=alluxio.heartbeat.HeartbeatContext --target=workers
 ```
+
+### Client-side Logging Configuration
+
+Often it's useful to change the logLevel of the Alluxio client running in the compute framework (e.g. Spark, Presto) process, and save it to a file for debugging. To achieve this, you can pass the following Java options to the compute 
+framework process.
+
+For example, the options `-Dalluxio.logs.dir=/var/alluxio/ -Dalluxio.logger.type=USER_LOGGER -Dlog4j.configuration=/tmp/
+alluxio/conf/log4j.properties` will instruct Alluxio client to use the log4j configuration in the Alluxio's conf path and 
```

Comment:
Doesn't `-Dlog4j.configuration` apply to all log4j loggers, not restricted to Alluxio?

Reply:
Right, but together with `-Dalluxio.logger.type=USER_LOGGER` and the following content in `alluxio/conf/log4j.properties`, it will write only client-side logging to the specified file:
```
# Appender for User
log4j.appender.USER_LOGGER=org.apache.log4j.RollingFileAppender
log4j.appender.USER_LOGGER.File=${alluxio.logs.dir}/user_${user.name}.log
log4j.appender.USER_LOGGER.MaxFileSize=10MB
log4j.appender.USER_LOGGER.MaxBackupIndex=10
log4j.appender.USER_LOGGER.layout=org.apache.log4j.PatternLayout
log4j.appender.USER_LOGGER.layout.ConversionPattern=%d{ISO8601} %-5p %c{1} - %m%n
```

Reply:
Will it interfere with the logging from the other applications or are log4j properties cumulative?

Reply:
This would interfere with the logging from the other applications, however, I feel it's acceptable since this is mainly for debugging. Alternatively, users can append these changes to the app's original `log4j` configuration file.

Reply:
sure